common/meta/api/src/share_api_test_suite.rs

```diff
@@ -286,33 +287,34 @@ impl ShareApiTestSuite {
 
         info!("--- add account account1 again");
         {
-            let req = AddShareAccountReq {
+            let req = AddShareAccountsReq {
                 share_name: share_name.clone(),
                 share_on,
-                account: account.to_string(),
+                if_exists,
+                accounts: vec![account.to_string()],
             };
 
-            let res = mt.add_share_account(req).await;
+            let res = mt.add_share_accounts(req).await;
             info!("add share account res: {:?}", res);
             let err = res.unwrap_err();
             assert_eq!(
-                ErrorCode::ShareAccountAlreadyExists("").code(),
+                ErrorCode::ShareAccountsAlreadyExists("").code(),
                 ErrorCode::from(err).code()
             );
         }
 
         info!("--- add account account2");
         {
-            let req = AddShareAccountReq {
+            let req = AddShareAccountsReq {
                 share_name: share_name.clone(),
                 share_on,
-                account: account2.to_string(),
+                if_exists,
+                accounts: vec![account2.to_string()],
```

Comment:
Should there be another test case for `if_exists`? Or this feature is included in other cases?